sdk/recoveryservices-siterecovery/Azure.ResourceManager.RecoveryServicesSiteRecovery/src/autorest.md

```diff
@@ -257,6 +257,7 @@ rename-mapping:
   RunAsAccount: SiteRecoveryRunAsAccount
   ServiceError: SiteRecoveryServiceError
   Severity: SiteRecoveryErrorSeverity
+  SiteRecoveryHealthError.innerHealthErrors: SiteRecoveryInnerHealthErrorsList
```

Comment:
This is the wrong json path, it should be `HealthError.innerHealthErrors`
refer to https://github.com/Azure/azure-rest-api-specs/blob/84eef1839d5db0edaf791fd0915b38a50b3b64b5/specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2025-01-01/service.json#L14078

Reply:
Fixed